l10n_br_account_product/models/account_invoice.py

```diff
@@ -777,20 +777,22 @@ def _compute_price(self):
     freight_value = fields.Float(
         'Frete', digits=dp.get_precision('Account'), default=0.00)
     fiscal_comment = fields.Text(u'Observação Fiscal')
-    xped = fields.Char(
+    customer_order = fields.Char(
         string=u"Código do Pedido (xPed)",
         size=15,
     )
-    nitemped = fields.Char(
+    customer_order_line = fields.Char(
```

Comment:
@renatonlima, considering that `account.invoice` models both customers and supplier invoices, maybe we should call this field `partner_order_code` and `partner_order_line` instead?

(sorry for not replying to [your comment](https://github.com/OCA/l10n-brazil/pull/299#discussion-diff-70673545) earlier...)